dags/common/parsing/generic_parsing.py

```diff
@@ -3,3 +3,7 @@ def take_first(arr):
         return next(filter(None, arr))
     except StopIteration:
         return None
+
+
+def list_to_value_dict(arr, key="value"):
+    return [{key: val} for val in arr]
```

Comment:
Should we check `if val`? in case the value is empty or None?

Reply:
Do we want {key: None} in our output ?

Reply:
No, we don't are anyway remove them https://github.com/SCOAP3/scoap3-next/blob/master/scoap3/modules/workflows/workflows/articles_upload.py#L473 I would say to ignore empty and None values

Reply:
Added check and test 👍